src/devices/Bmp180/samples/Bmp180.Sample.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <LangVersion>7.3</LangVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <PlatformTarget>AnyCPU</PlatformTarget>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease.19080.1" />
+    <PackageReference Include="System.Devices.Gpio" Version="0.1.0-preview2-181222-2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Bmp180">

```

Comment:
Instead of referencing to the .dll directly, add a ProjectReference similar to what we do here:

[https://github.com/dotnet/iot/blob/0a4a036ef8404aa68b9437655690679bd53358c0/src/devices/Mcp23xxx/samples/Mcp23xxx.Samples.csproj#L12-L14](https://github.com/dotnet/iot/blob/0a4a036ef8404aa68b9437655690679bd53358c0/src/devices/Mcp23xxx/samples/Mcp23xxx.Samples.csproj#L12-L14)